Tools/autotest/sim_vehicle.py

```diff
@@ -1267,6 +1269,10 @@ def generate_frame_help():
                      default=False,
                      action='store_true',
                      help="Set SYSID_THISMAV based upon instance number")
+group_sim.add_option("", "--sim-address",
```

Comment:
why adding extra global parameter here.
you can receive your own parameters by using --model webots-python:127.0.0.1:1234 ...etc.

Reply:
I think ArduPilot should have the ability regardless as it is a parameter of SITL_cmdline.cpp. For my specific case this allows custom IP for use with WSL2 for example. However, it should also work the same way with Gazebo and other implementations as they accept an IP from `set_interface_ports`.